passage/models.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, layers, cost, updater='Adam', verbose=2, Y=T.matrix(), iterat
         self._cost = theano.function([self.X, self.Y], cost)
         self._predict = theano.function([self.X], self.y_te)
 
-    def fit(self, trX, trY, batch_size=64, n_epochs=1, len_filter=LenFilter()):
+    def fit(self, trX, trY, batch_size=64, n_epochs=1, len_filter=LenFilter(), snapshot_freq=1, path=None):
```

Comment:
This function could probably do with a docstring generally explaining each of the inputs/outputs. PEP257 compliance would be awesome, but I don't think it's required. @Newmu, thoughts?